### PR TITLE
Improved User Experience

### PR DESCRIPTION
**Pull request by Naatan**

### CODE REVIEW

File: cmd/state/errors.go [hunk 1]
```diff
@@ -5,17 +5,77 @@ import (
...
+	}
+	return "\n" + strings.Join(outLines, "\n")
+}
+
+func trimError(msg string) string {
+	if strings.Count(msg, ".") > 1 || strings.Count(msg, ",") > 1 {
+		return msg // Don't trim dots if we have multiple sentences.
+	}
+	return strings.TrimRight(msg, " .")
+}
```
Comment (mdrohmann) [inline cmd/state/errors.go:77]: I don't quite understand what the `trimError` function is supposed to do. 
Could you please explain?

 I also noticed that it is currently skipping the trimming when there are at least *2* dots or commas in the message.   Should it be *1* instead?

**Review by mdrohmann — CHANGES_REQUESTED** [review]
It looks pretty cool.

File: cmd/state/errors.go (hunk 1 above)
Comment (Naatan) [reply cmd/state/errors.go:77]: This is making the errors consistently not have a period at or whitespace at the end UNLESS the sentence includes a comma or multiple dots (ie. is more than one sentence).

Comment (mdrohmann) [reply cmd/state/errors.go:77]: Should it then be: ``` strings.Count(msg, ",") > 0 ``` ?

File: cmd/state/errors.go [hunk 2]
```diff
@@ -5,17 +5,77 @@ import (
...
+	// Print tips
+	outLines = append(outLines, output.Heading(locale.Tl("err_more_help", "Need More Help?")).String())
+	for _, tip := range errorTips {
+		outLines = append(outLines, fmt.Sprintf(" [DISABLED]-[/RESET] %s", trimError(tip)))
+	}
+	return "\n" + strings.Join(outLines, "\n")
+}
+
+func trimError(msg string) string {
+	if strings.Count(msg, ".") > 1 || strings.Count(msg, ",") > 1 {
```
Comment (Naatan) [inline cmd/state/errors.go:73]:
```suggestion
	if strings.Count(msg, ".") > 1 || strings.Count(msg, ",") > 0 {
```

File: cmd/state/errors.go (hunk 1 above)
Comment (Naatan) [reply cmd/state/errors.go:77]: Yes, you're right! fixed.

**Review by mdrohmann — APPROVED** [review]

**Pull request merged**